benchmarks/index.js

```diff
@@ -0,0 +1,34 @@
+var DataSource = require('loopback-datasource-juggler').DataSource;
+var connector = require('..');
+var Benchmark = require('benchmark');
+
+var ds = new DataSource(connector, {
```

Comment:
@raymondfeng Is this the right way to initialize a datasource manually? I'm starting to think this is why it doesn't disconnect.

Maybe it should be:
```
var ds = new DataSource('mongodb', opts)
var connector = new Connector(ds, cb);
```

I noticed because looking https://github.com/strongloop/loopback-connector-mongodb/blob/master/lib/mongodb.js#L69-L71, it seems like we only connect if a callback is provided.